homework-submission/week1/src/components/List.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from "react";
+
+
+class List extends Component {
+    render() {
+        return (
+            <div>
+                <li>{this.props.description}, {this.props.deadline} </li>
```

Comment:
you are inserting an `li`inside of a `div` tag, from the [documentations](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li) we will notice that the Permitted parents fro an element are very specific, it's okay for an example but in a real world it may cause a bugs.